core/src/types.ts

```diff
@@ -84,14 +84,37 @@ export declare interface UniswapV2LpTokenCalleeConfig {
     token1: string;
 }
 
+interface ValueSlotAddressAndOffset {
+    slot: string;
+    offset: number;
+}
+
 export declare interface CollateralConfig {
     title: string;
     ilk: string;
     symbol: string;
     decimals: number;
     exchange: RegularCalleeConfig | UniswapV2LpTokenCalleeConfig;
+    oracle: CollateralPriceSourceConfig;
+}
+
+interface OracleConfigBase {
+    hasDelay: boolean;
+    slotPriceValueBeginsAtPosition: number;
+}
```

Comment:
- [ ] Why `currentPriceSlotAddress` is not part of the `OracleConfigBase` if it's part of the both `OracleCurrentAndNextPrices` and `OracleCurrentPriceOnly`?

- [ ] Is it now just 2 types, not 3?

Reply:
> Is it now just 2 types, not 3?